bigtable/tests/instance_admin_integration_test.cc

```diff
@@ -47,6 +47,18 @@ class InstanceAdminIntegrationTest : public ::testing::Test {
 bool UsingCloudBigtableEmulator() {
   return std::getenv("BIGTABLE_EMULATOR_HOST") != nullptr;
 }
+bool IsInstancePresent(
+    std::vector<::google::bigtable::admin::v2::Instance> instances,
+    std::string instance_name) {
+  bool instance_present = false;
+  for (auto const& i : instances) {
+    if (instance_name.compare(i.name()) == 0) {
```

Comment:
I think `instance_name == i.name()` is more natural in this case.

Reply:
fixed.